app/src/main/res/menu/home_menu.xml

```diff
@@ -50,6 +50,10 @@
       android:title="@string/menu_jump"
       app:showAsAction="never" />
   <item
+      android:id="@+id/translations"
+      android:title="Translations"
```

Comment:
hey so ive looked at this a bit, focusing on TranslationsSpinnerAdapter and PagerActivity. I'm running into issues finding a straightforward solution -  the translation spinner has a checkbox dropdown item, and implementing another view to the dropdown such as TextView is non-trivial unless both items are added to a parent view. Additionally, since the spinner has checkboxes, an event listener for OnItemSelectedListener isn't being picked up if implemented in PagerActivity.  Please let me know if you have some guidance on this. Thank you